release/src/ShopwarePluginRelease.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Release;
+
+use Bramus\Monolog\Formatter\ColoredLineFormatter;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+
+class ShopwarePluginRelease
+{
+    /** @var LoggerInterface */
+    private $logger;
+
+    /** @var ReleaseBundler */
+    private $releaseBundler;
+
+    /** @var ReleaseTagger */
+    private $releaseTagger;
+
+    public function __construct(
+        LoggerInterface $logger,
+        ?ReleaseBundler $releaseBundler = null,
+        ?ReleaseTagger $releaseTagger = null
+    ) {
+        $this->logger = $logger;
+        $this->releaseBundler = $releaseBundler ?? new ReleaseBundler($logger);
+        $this->releaseTagger = $releaseTagger ?? new ReleaseTagger($logger);
+    }
+
+    public static function buildLogger(string $name): LoggerInterface
```

Comment:
can be private then.

Reply:
See https://github.com/findologic/plugin-shopware-6/pull/184#discussion_r710821486

Reply:
As discussed, move to Utils.